### PR TITLE
[feat] Add SDK utils tests

### PR DESCRIPTION
**Pull request by yeghiakoronian**
- aim-add-tests-to-sdk-utils
- remove a test, seems its making conflict with another test

### CODE REVIEW

**Comment by mihran113** [comment]
Hey @yeghiakoronian. Thanks a lot for the effort. 
Can I ask you to change the PR headline to `[feat] Add SDK utils tests` to match our guidelines?

File: tests/sdk/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+        # act
+        path, found = utils.search_aim_repo(path)
+
+        # assert
+        self.assertFalse(found)
+        self.assertIsNone(path)
+
+    @mock.patch('os.path.exists')
+    def test_should_search_aim_repo_not_found_dot_back_sash(self, mock_os_path_exists: mock.MagicMock):
```
Comment (alberttorosyan) [inline tests/sdk/test_utils.py:25]: same as previous comment

**Review by alberttorosyan — CHANGES_REQUESTED** [review]
hey @yeghiakoronian ! thanks for the contribution, looks great!
left a small comment wrt typos.

**Comment by yeghiakoroniannexonia** [comment]
@alberttorosyan all done

File: tests/sdk/test_utils.py (hunk 1 above)
Comment (yeghiakoronian) [reply tests/sdk/test_utils.py]: @alberttorosyan done

**Review by alberttorosyan — APPROVED** [review]

**Comment by alberttorosyan** [comment]
@yeghiakoronian may I ask to also update `CHANGELOG.md`?
We need to automate the changelog generation yet, meanwhile each change is added manually.

**Pull request merged**